keps/sig-instrumentation/20190605-metrics-stability-migration.md

```diff
@@ -105,26 +105,33 @@ The [metrics overhaul KEP](https://github.com/kubernetes/enhancements/blob/maste
 
 ## Implementation History
 
-- [x] [Migrate kubelet metrics to use standard prometheus collectors](https://github.com/kubernetes/kubernetes/issues/79286)
-- [ ] Create migrated variants of shared client metrics
-- [ ] Create migrated variants of shared leader-election metrics
-- [ ] Create migrated variants of shared workqueue metrics
-- [ ] Migrate kubelet's /metrics/probes endpoint
-- [ ] Migrate apiserver /metrics endpoint
-- [ ] Migrate scheduler /metrics endpoint
-- [ ] Migrate kube-proxy /metrics endpoint
-- [ ] Migrate controller-manager /metrics endpoint (this include in-tree cloud-provider metrics)
-
-TBD (since this is not yet implemented)
+- [x] [Migrate kubelet metrics to use standard prometheus
+  collectors](https://github.com/kubernetes/kubernetes/issues/79286)
+- [x] Create migrated variants of shared client metrics
+- [x] Create migrated variants of shared leader-election metrics
+- [x] Create migrated variants of shared workqueue metrics
+- [x] [Migrate kubelet's /metrics/probes
+  endpoint](https://github.com/kubernetes/kubernetes/pull/81534)
+- [x] [Migrate apiserver /metrics
+  endpoint](https://github.com/kubernetes/kubernetes/pull/81531)
+- [x] [Migrate scheduler /metrics
+  endpoint](https://github.com/kubernetes/kubernetes/pull/81576)
+- [x] [Migrate kube-proxy /metrics
+  endpoint](https://github.com/kubernetes/kubernetes/pull/81626)
+- [x] [Migrate controller-manager /metrics
+  endpoint](https://github.com/kubernetes/kubernetes/pull/81624) (this includes
+  in-tree cloud-provider metrics)
 
 ## Graduation Criteria
 
-- [ ] Prior to migrating a component, automated static analysis testing is in place to validate and verify API guarantees.
+- [x] Prior to migrating a component, [automated static analysis testing is in
+  place](https://github.com/kubernetes/kubernetes/pull/81510) to validate and
+  verify API guarantees.
 - [ ] Adequate documentation exists for new flags on components.
 - [ ] Update instrumentation documents to reflect changes
```

Comment:
Not sure if I should be marking this one as `implemented` given that these check boxes still haven't been checked...

Reply:
They have documentation strings, as all command line flags do. The docs should be auto-generated to reflect those documentation strings.